src/main/java/org/popcraft/popcraft/commands/Music.java

```diff
@@ -1,54 +1,68 @@
 package org.popcraft.popcraft.commands;
 
+import com.google.common.base.Joiner;
 import org.bukkit.ChatColor;
 import org.bukkit.Sound;
 import org.bukkit.command.Command;
 import org.bukkit.command.CommandExecutor;
 import org.bukkit.command.CommandSender;
 import org.bukkit.entity.Player;
+import org.popcraft.popcraft.newCode.PopCommand;
 import org.popcraft.popcraft.utils.Message;
 
 import java.text.DecimalFormat;
 import java.util.HashMap;
+import java.util.Map;
 import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
-@Deprecated
+@PopCommand("music")
 public class Music implements CommandExecutor {
 
-    private static HashMap<UUID, Sound> playing = new HashMap<UUID, Sound>();
+    private static final Map<String, Sound> DISCS = Stream.of(Sound.values()).filter(item -> item.toString().startsWith("RECORD_")).collect(Collectors.toMap(item -> item.name().replace("RECORD_", "").toLowerCase(), item -> item));
+    private static Map<UUID, Sound> playing = new HashMap<>();

```

Comment:
Eventually we should change these to be thread safe. Collections.synchronizedMap(new HashMap<>());